docs/process.md

```diff
@@ -26,6 +26,9 @@
       should pass after each commit).
    When landing multiple commits in such a scenario, use the "rebase" option,
    to avoid a merge commit.
+ * Add `NFC` to the end of the PR title for Non-Function Changes.
+ * Add a `[prefix]` to start of the PR title to signify the subsystem or area
+   that the PR targets. e.g. `[test] Update foo test` or `[ports] Fix zlib port`
```

Comment:
In this case the PR seems to more suggest what we should do rather than document existing practices :smile: but sgtm!

Reply:
We have done it sometimes.. but inconsistently.  e.g. `foo:` vs `[foo]`